src/components/directory-content.js

```diff
@@ -652,6 +653,60 @@ const DirectoryContent = () => {
         return undefined;
     };
 
+    function buildFilesToDeleteGrid(files) {
```

Comment:
must be managed inside the deleteDialog component not in directory-content

Reply:
done